cpp/include/cuspatial/detail/point_linestring_nearest_points.cuh

```diff
@@ -54,7 +54,7 @@ template <class Vec2dItA,
           class OffsetIteratorB,
           class OffsetIteratorC,
           class OutputIt>
-void __global__
+void CUSPATIAL_KERNEL
```

Comment:
Let's swap these to match all other kernels?

Reply:
Done